src/js/components/Posts.js

```diff
@@ -1,103 +1,95 @@
-                <div className="mainContainer">
-                    <header>
-                        <h2>Sharing our journey and work with the world</h2>
-                    </header>
-                    <div className="columns">
-                        <section>
-                            <h3 className="title">Our journey</h3>
-                            <div className="postColumn" id="non-tech-posts">
-                                <Posts kind={NON_TECH} errorHandler={this.notifyError} />
-                            </div>
-                        </section>
-                        <section>
-                            <h3 className="title">Technical posts</h3>
-                            <div className="postColumn" id="tech-posts">
-                                <Posts kind={TECH} errorHandler={this.notifyError} />
-                            </div>
-                        </section>
-                    </div>
-                </div>
-            </aside>
-        )
-    }
-}
-
-class Posts extends Component {
-    constructor(props) {
-        super(props)
-        this.state = {
-            state: states.LOADING,
-            posts: []
-        }
-    }
+    loadPosts(category)
+      .then(posts => this.setState({ posts }))
+  }
 
-    componentWillMount() {
-        const loader = this.props.kind == TECH ? loadTechPosts : loadNonTechPosts
-        loader()
-            .then(posts => this.setState({ state: states.LOADED, posts }))
-            .catch(err => {
-                console.error(err)
-                this.setState({ state: states.ERROR })
-                this.props.errorHandler(this.props.kind)
-            })
+  render() {
+    const {posts, category, title, moreText} = this.state
+    if (posts.length == 0) {
+      return <div></div>
     }
 
-    render() {
-        const { state, posts } = this.state
-        if (state === states.LOADING) {
-            return <Loading />
-        } else if (state === states.ERROR) {
-            return <LoadingError />
-        }
+    document.dispatchEvent(events.ACTIVE_EVENT);
 
-        return (
-            <div className='posts'>
-                {posts.map((p, i) => <Post key={i} first={i === 0} post={p} />)}
-            </div>
-        )
-    }
+    return (
+      <div>
+          <h2>{title}</h2>
+          <div className="cards">
+              {posts.map((p, i) => <Post key={i} first={i === 0} post={p} />)}
+          </div>
+          <div className="blog__category__wrapper">
+              <a href={blogUrl('categories/' + category)} target="_blank" className="btn-pill">
+                  {moreText}
+              </a>
+          </div>
+      </div>
+    )
+  }
 }
 
 function Post({ post, first }) {
+  return (
+    <article className="cards__element cards__element_blog">
+        <a href={post.link} target="_blank">
+            <header>
+                <figure style={{background: 'linear-gradient(to right, #36536C, #2377B6)'}}>
```

Comment:
I'll parametrice it, because this border is a different gradient than the mixin one...